views/_includes/backButton.pug

```diff
@@ -1,2 +1,2 @@
 if prevRoute
```

Comment:
Ah, is this why it doesn't show up sometimes?

Looks like 
- it's not on the start page (which we want)
- it's not on a page with an error (which we don't want ☹️)

See if you can figure out why it doesn't show up on the `/login/code` page when there's an error.